### DISCUSSION TITLE
What version should I use for now?

### DISCUSSION THREAD

**Discussion by riobard** [memtest86plus/memtest86plus#176] [Q&A] [answered]
I'm a bit puzzled… On the front page the only possible download is v6 with a big red alert saying it's experimental and not for production, and on the Archives page every version is obsolete and not supported. 

So what am I supposed to use now?

**Comment by debrouxl** [comment, marked as answer]
See also #142 :)
But real life gets in the way of plans.

**Reply by riobard** [reply]
hmmm… this is my first encounter of deprecating old versions _before_ a new stable version is ready 😂

**Reply by x86fr** [reply]
Everything is almost ready for v6 release. There is a last patch to add on my side and it will be released.

I'm moving to a new house right now and all my stuff is packed up. As soon as I get my Internet connection back (planned next Friday), I'll start the release process.

**Reply by martinwhitaker** [reply]
I'd like PR #167 to be merged before the release, if we think it's had enough testing.

**Reply by 01e3** [reply]
If it helps, I was able to test https://github.com/memtest86plus/memtest86plus/pull/167 and confirmed it [works](https://github.com/memtest86plus/memtest86plus/pull/167#issuecomment-1264515095)

**Comment by Fantu** [comment]
@x86fr @martinwhitaker hi, is there any news about 6.00 release?

**Reply by x86fr** [reply]
Next week. I'm back online since Friday.

**Comment by x86fr** [comment]
Memtest86+ v6.00 final has been released
